2022-08-09 06:40:01 - loading data from datasets/yelp/train.json
  0%|          | 0/30400 [00:00<?, ?it/s]100%|██████████| 30400/30400 [00:00<00:00, 2106123.81it/s]
2022-08-09 06:40:01 - loading data from datasets/yelp/valid.json
  0%|          | 0/3800 [00:00<?, ?it/s]100%|██████████| 3800/3800 [00:00<00:00, 2069378.76it/s]
2022-08-09 06:40:01 - loading data from datasets/yelp/test.json
  0%|          | 0/3800 [00:00<?, ?it/s]100%|██████████| 3800/3800 [00:00<00:00, 2040240.04it/s]
2022-08-09 06:40:01 - loading features from datasets/yelp/train_bert.pkl
2022-08-09 06:40:01 - loading features from datasets/yelp/valid_bert.pkl
2022-08-09 06:40:01 - loading features from datasets/yelp/test_bert.pkl
Traceback (most recent call last):
  File "fwrench/applications/pipeline.py", line 259, in <module>
    fire.Fire(main)
  File "/hdd2/kaylee/anaconda3/envs/AutoWS-Bench-101/lib/python3.7/site-packages/fire/core.py", line 141, in Fire
    component_trace = _Fire(component, args, parsed_flag_args, context, name)
  File "/hdd2/kaylee/anaconda3/envs/AutoWS-Bench-101/lib/python3.7/site-packages/fire/core.py", line 471, in _Fire
    target=component.__name__)
  File "/hdd2/kaylee/anaconda3/envs/AutoWS-Bench-101/lib/python3.7/site-packages/fire/core.py", line 681, in _CallAndUpdateTrace
    component = fn(*varargs, **kwargs)
  File "fwrench/applications/pipeline.py", line 137, in main
    train_data_embed = embedder.transform(train_data)
  File "/hdd2/kaylee/AutoWS-Bench-101/fwrench/embeddings/openai_clip_embedding.py", line 79, in transform
    y = classes_[self.dataset]
KeyError: 'yelp'
